docs/toc.md

```diff
@@ -97,6 +97,7 @@
 
 # [.NET Core Guide](core/index.md)
 ## [Getting started](core/getting-started.md)
+## [PreRequisites](core/prerequisites.md)
```

Comment:
The article uses "Prerequisites", the title "Pre-Requisites". I guess both are okay, though it would be better to use one of them consistently. But I don't think "PreRequisites", as used here, is correct.